src/preview/viewer.ts

```diff
@@ -105,24 +95,19 @@ async function view(pdfFile: string, mode?: 'tab' | 'browser' | 'external'): Pro
  *
  * @param pdfFile The path of a PDF file.
  */
-async function viewInBrowser(pdfFile: string): Promise<void> {
-    const url = await getUrl(pdfFile)
-    if (!url) {
-        return
-    }
-    const pdfUri = vscode.Uri.file(pdfFile)
-    manager.create(pdfUri)
-    lw.watcher.pdf.add(pdfUri)
+async function viewInBrowser(pdfFile: vscode.Uri): Promise<void> {
+    manager.create(pdfFile)
+    lw.watcher.pdf.add(pdfFile)
     try {
-        logger.log(`Serving PDF file at ${url}`)
-        await vscode.env.openExternal(vscode.Uri.parse(url, true))
-        logger.log(`Open PDF viewer for ${pdfUri.toString(true)}`)
+        logger.log(`Serving PDF file at ${pdfFile.fsPath}`)
+        await vscode.env.openExternal(pdfFile)
+        logger.log(`Open PDF viewer for ${pdfFile.toString(true)}`)
     } catch (e: unknown) {
         void vscode.window.showInputBox({
             prompt: 'Unable to open browser. Please copy and visit this link.',
-            value: url
+            value: pdfFile.fsPath
```

Comment:
Behavior changed. Originally it is a url to the web viewer. Now it becomes the pdf path.